bin/test-lxd-storage-vm

```diff
@@ -420,6 +420,56 @@ do
         lxc image delete vmbig
         lxc profile delete vmsmall
 
+        echo "==> Checking VM UUID while snapshotting (non-stateful mode)"
+        lxc init images:ubuntu/20.04/cloud v1 --vm -s "${poolName}"
+        lxc start v1
+        waitVMAgent v1
+        lxc info v1
+
+        initialUUID=$(lxc config show v1 --expanded | grep "volatile.uuid:" | awk '{print $2}')
+        lxc snapshot v1 snap1
+        lxc restore v1 snap1
+        newUUID=$(lxc config show v1 --expanded | grep "volatile.uuid:" | awk '{print $2}')
+
+        if [ "${initialUUID}" = "${newUUID}" ]; then
+                echo "==> UUID of the VM instance should be different after restoring its snapshot"
+                false
+        fi
+
+        echo "==> Checking VM UUID while snapshotting (stateful mode)"
+        lxc launch images:busybox/1.34.1 v2 --vm -s "${poolName}" --ephemeral
```

Comment:
Please use `images:ubuntu/jammy` for VM tests.

You only need to use busybox for jenkins test suite tests and/or container stateful tests.
But this test is only for VMs so we can use a normal Ubuntu image.